azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -349,6 +357,16 @@ func resourceArmKeyVaultCertificateRead(d *schema.ResourceData, meta interface{}
 	if contents := cert.Cer; contents != nil {
 		d.Set("certificate_data", string(*contents))
 	}
+
+	x509Thumbprint, err := base64.RawURLEncoding.DecodeString(string(*cert.X509Thumbprint))
+	if err != nil {
+		return err
+	}
+
+	x509ThumbprintHex := hex.EncodeToString(x509Thumbprint)
+
+	d.Set("thumbprint", strings.ToUpper(x509ThumbprintHex))
```

Comment:
very minor, but we could combine these two lines as such: `d.Set("thumbprint", strings.ToUpper(hex.EncodeToString(x509Thumbprint)))`